test_integration/m1_facilitator/00_deployment/deployment.test.ts

```diff
@@ -0,0 +1,95 @@
+// Copyright 2020 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Web3 from 'web3';
+import Mosaic from 'Mosaic';
+
+import { AbiItem } from 'web3-utils';
+import { Anchor } from 'Mosaic/dist/interacts/Anchor';
+import { ERC20Gateway } from 'Mosaic/dist/interacts/ERC20Gateway';
+import { UtilityToken } from 'Mosaic/dist/interacts/UtilityToken';
+import { Gen0ERC20Cogateway } from 'Mosaic/dist/interacts/Gen0ERC20Cogateway';
+import docker from '../docker';
+import shared from '../shared';
+import Utils from '../utils';
+
+describe('Deployment of contract ', () => {
```

Comment:
eslint error: `Missing return type on function.eslint(@typescript-eslint/explicit-function-return-type`

Reply:
Please fix this error in other tests also.

Reply:
Done